build.gradle

```diff
@@ -1,37 +1,19 @@
-buildscript {
-    repositories {
-        mavenLocal()
-        maven { url "https://nexus.ala.org.au/content/groups/public/" }
-        maven { url "https://repo.grails.org/grails/core" }
-    }
-    dependencies {
-        classpath "org.grails:grails-gradle-plugin:$grailsVersion"
-        classpath "org.grails.plugins:hibernate5:${gormVersion}"
-        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:3.4.6"
-        classpath "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:$webdriverBinariesVersion"
-        classpath 'org.grails.plugins:database-migration:4.2.0'
-    }
-}
-
 plugins {
+    id "groovy"
+    id "org.grails.grails-web"
+    id "org.grails.grails-gsp"
+    id "application"
+    id "war"
+    id "maven-publish"
+    id "com.bertramlabs.asset-pipeline"
     id "com.gorylenko.gradle-git-properties" version "2.4.1"
 }
 
-version "4.2.2-SNAPSHOT"
+version "4.3-SNAPSHOT"
```

Comment:
Move version and group to the top under `buildscript` e.g. https://github.com/AtlasOfLivingAustralia/collectory/blob/develop/build.gradle#L1

Reply:
Fixed, committed .